contributing/code/patches.rst

```diff
@@ -113,7 +113,9 @@ Choose the right Branch
 Before working on a patch, you must determine on which branch you need to
 work:
 
-* ``2.3``, if you are fixing a bug for an existing feature or want to make a
+* ``2.3``, for security issues;
```

Comment:
technically, you should not be working on security issues in the open. Work on security issues happens through a different process than the one described in this documentation page (due to being handled privately)